onnxruntime/core/providers/dml/DmlExecutionProvider/src/DmlGraphFusionTransformer.cpp

```diff
@@ -0,0 +1,391 @@
+#pragma once
+
+#include "precomp.h"
+#include "GraphDescBuilder.h"
+#include "ExecutionProvider.h"
+#include "DmlGraphFusionTransformer.h"
+#include "GraphPartitioner.h"
+#include "core/framework/kernel_type_str_resolver.h"
+#include "core/framework/kernel_lookup.h"
+#include "FusedGraphKernel.h"
+#include "MLOperatorAuthorImpl.h"
+#include "GraphKernelHelper.h"
```

Comment:
>GraphKernelHelper [](http://example.com/codeflow?start=10&length=17)

The couple methods used from this file can be move here

Reply:
GraphKernelHelper::GetInputConstness is also used by FusedGraphKernel.cpp while ProcessInputData. I have moved the other one (GraphKernelHelper::ConvertGraphDesc) here.